packages/test/test-service-load/src/gcDataStores.ts

```diff
@@ -197,26 +194,37 @@ export class DataObjectNonCollab extends BaseDataObject implements IGCDataStore
     public async run(config: IRunConfig, id?: string): Promise<boolean> {
         console.log(`########## Started child [${id}]`);
         assert(config.testConfig.inactiveTimeoutMs !== undefined, "inactive timeout is required for GC tests");
+
+        this.myId = id;
+        this.shouldRun = true;
         // Set the local inactive timeout 500 less than the actual to keep buffer when marking data stores as inactive.
         this._inactiveTimeoutMs = config.testConfig.inactiveTimeoutMs - 500;
-        // Set up the child to send ops opRateMultiplierPerLevel times faster than this data store.
-        const opRatePerMin = config.testConfig.opRatePerMin * opRateMultiplierPerLevel;
+
+        /**
+         * Adjust the totalSendCount and opRatePerMin. There can be maximum of maxRunningLeafChildren children running
+         * at the same time.
+         * - This data store and each child sends 1/4th of the number of ops per min.
+         * - This data store sends 1/4th the number of total ops since its sending at 1/4th the op rate.
+         */
+        const opRatePerMin = Math.ceil(config.testConfig.opRatePerMin / (maxRunningLeafChildren + 1));
+        const totalSendCount = config.testConfig.totalSendCount / 4;
```

Comment:
Shouldn't `4` be replaced with `(maxRunningLeafChildren + 1)`

Reply:
And update the comment above to not say 1/4 like it's always 4

Reply:
Ohh... max is const 3 so they're equivalent.  Just being consistent in the code would be good.

Reply:
you are right. It should be maxRunningLeafChildren + 1.